src/vaadin-text-area.html

```diff
@@ -183,8 +204,10 @@
           this._oldValueLength = valueLength;
 
           const inputHeight = input.scrollHeight;
-          if (inputHeight > input.clientHeight) {
+          if (inputHeight > input.clientHeight && !this.rows) {
             input.style.height = inputHeight + 'px';
+          } else if (this.rows) {
+            input.style.height = 'initial';
```

Comment:
Looks like a `<vaadin-text-area>` with `rows` doesn't scroll when the content is long enough to overflow. This is due to the explicit `overflow: hidden` on the `<textarea>` which should probably not be in use when `rows` is defined.